### ISSUE TITLE
tf.losses.* namespace lacks the losses from tfjs-layers

### ISSUE THREAD

**Issue by caisq** [tensorflow/tfjs#1614]
#### TensorFlow.js version
1.1.2

The public API namespace tf.losses.* of tfjs (union package) lacks all the loss functions defined in tfjs-layers, such as `categoricalCrossentropy`, `binaryCrossentropy` and so forth. Instead, it contains only four loss functions defined in tfjs-core.

The two sets of loss functions should be combined into one and exposed in the union package.

**Comment by gaikwadrahul8** [comment]
Hi, @caisq 

Apologize for the delayed response and we have updated our TFJS API official documentation w.r.t [Losses](https://js.tensorflow.org/api/4.4.0/#Training-Losses:~:text=tf.train.rmsprop-,Losses,tf.losses.softmaxCrossEntropy,-Classes) and I see `categoricalCrossentropy` and `binaryCrossentropy` losses are part of Metrics section [here](https://js.tensorflow.org/api/4.4.0/#Training-Losses:~:text=tf.browser.toPixels-,METRICS,tf.metrics.sparseCategoricalAccuracy,-CALLBACKS) so that's what you were looking or all the losses under one section only ? If have I missed something please let me know ? Thank you!
